tests/example.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { test, expect } from '@playwright/test';
+
+const productCardByName = 'h5[data-test="product-name"]';
+
+test('Test 1: Verify login with valid credentials', async ({ page }) => {
+  await page.goto(process.env.WEB_URL + '/auth/login');
```

Comment:
Since we have base url set in playwright.config.ts, we dont need to specify a full path all the time. Relative path should work fine.
```suggestion
  await page.goto('/auth/login');
```